Docs: Update streaming datasource tutorial (#2516)
grafana/plugin-tools: docusaurus/docs/tutorials/build-a-streaming-data-source-plugin.md

      return getGrafanaLiveSrv().getDataStream({
        addr: {
          scope: LiveChannelScope.DataSource,

>           stream: this.uid, // since the scope is a data source the UID should be used

Reviewer: ```suggestion
          stream: this.uid, // since the scope is a data source, use the UID
```
